docs/faq-and-troubleshooting/common-questions/roblox-not-opening.md

```diff
@@ -5,35 +5,37 @@ icon: simple/roblox
 
 # :simple-roblox: Why is Roblox not opening?
 
-**You may be missing TPM 2.0 and Secure Boot. You can also make a new user account and move everything over.**
```

Comment:
It's best to keep a bolded point but change it to something like **Some users may experience Roblox not opening on Atlas due to various factors.**